modules/smart-agent_cassandra-nodetool/conf/01-state.yaml

```diff
@@ -0,0 +1,14 @@
+## Example
+module: cassandra nodetool
+name: "Node is not in normal status"
+
+transformation: ".min(over='10m')"
```

Comment:
I am not sure of the time it can take for a node to join/leave "normally" a cluster.

may be it is safer to increase a bit this timeframe?

Reply:
Also, a single node down is not as much problematic than the cluster down.

here there is no cluster aggregation by default so I would set a long value here because:
- a member in abnormal state during long time is ALWAYS a problem (>= 1hour I think)
- a member in abnormal state during few time is ALWAYS NOT a problem